host/testcluster.go

```diff
@@ -27,10 +27,8 @@ import (
 	"testing"
 	"time"
 
-	"github.com/uber/cadence/common/log/loggerimpl"
-
+	"github.com/startreedata/pinot-client-go/pinot"
```

Comment:
nit: import groups should ideally go in following order
- builtin packages
- repo specific packages
- external packages
- named packages

Reply:
I'm using default goimports behaviour. 
As far as I've seen discussions about grouping, the outcome is that there should be either 2 or 3 groups. But in Uber style guide, we've settled in 2 groups. https://github.com/uber-go/guide/blob/master/style.md#import-group-ordering

Reply:
broadly I agree, but tbh I don't think it matters much, and it doesn't affect behavior as of [some semi-recent Go version].  automate it or ignore it, imo, it's hidden in almost all editors anyway and it doesn't make code review any more difficult.

Reply:
It's still a bit surprising to me that goimports doesn't have an option to forcibly reorder stuff... but this looks like it might be reasonable: https://github.com/incu6us/goimports-reviser

I might see if that does what we want.  The less we have to touch here, the better.

Reply:
it's like 99% good... but it strips comments on anything it moves :|
if that's fixed it seems quite promising tho.  probably not too hard, astutil.CommentMap addresses exactly this kind of thing.

Reply:
I'd vote to put whatever behaviour we settle on in a linter and just enforce it at the lint level

Reply:
There is also this https://github.com/NonLogicalDev/gofancyimports which is pretty good. It is metioned in a issue for goimports https://github.com/golang/go/issues/20818, though it is quite fresh.

Unfortunately that even in monorepo there is no tooling to enforce proper imports.

Reply:
gofancyimports looks great at a glance:
- keeps comments
- enforces grouping
- drops `import ( "single/thing" )` parenthesis
- comments above an import keeps that group of imports organized together (manual grouping when desirable)
- comments on the same line as an import get reordered with other imports

I can get a PR for that up today, it's pretty simple.  just gotta verify the changes/license/etc first.